drag_and_drop_v2/public/js/view.js

```diff
@@ -50,13 +51,21 @@
         if (item.is_placed) {
             style.left = item.x_percent + "%";
             style.top = item.y_percent + "%";
+            tabindex = -1;  // If an item has been placed it can no longer be interacted with,
+                            // so remove the ability to move focus to it using the keyboard
         }
         return (
             h('div.option',
                 {
                     key: item.value,
                     className: item.class_name,
-                    attributes: {'data-value': item.value, 'data-drag-disabled': item.drag_disabled},
+                    attributes: {
+                        'tabindex': tabindex,
+                        'draggable': true,
+                        'aria-grabbed': false,
```

Comment:
@itsjeyd Instead of manually toggling the `aria-grabbed` attribute in drag start and stop event handlers, can we instead set a flag in the state and set the attribute here based on that flag? Something like `state.items[item_id].grabbed = true; applyState()` inside the `start` handler and then `attributes: {'aria-grabbed': item.grabbed, ....}` here in the view function.

It's true that during the drag the virtual dom representation doesn't exactly match the actual dom (the position of the dragged item is controlled by jquery dnd plugin rather than virtual dom while the drag is in progress), but it still makes sense to try and minimize manual dom manipulation as much as possible.